### PR TITLE
[ID] Add `Play style/Double tapping`

### PR DESCRIPTION
**Pull request by Fhoxydeo**

### CODE REVIEW

File: wiki/Play_style/Double_tapping/id.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+stub: true
```
Comment (TicClick) [inline wiki/Play_style/Double_tapping/id.md:2]: the original isn't a stub anymore

File: wiki/Play_style/Double_tapping/id.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Penggunaan
+
+Double tapping biasanya menggantikan gaya tapping utama pemain karena mudah digunakan dan meningkatkan "kecepatan" (meskipun pemain sebenarnya tidak mengetuk lebih cepat). Cara untuk mendapatkan kecepatan yang lebih besar ini dapat dicapai dengan memanfaatkan hit window yang bertumpukan, muncul keseringan pada not-not bertumpuk dan stream. Jadi, satu input tombol terhitung sebagai satu hit circle dalam stack, lingkaran berikutnya juga akan tercatat sebagai hit karena hit window mereka sangat berdekatan sehingga saling tumpang tindih. Tetapi sebagai akibatnya, hampir tidak mungkin untuk mengontrol waktu tapping ini yang dapat mengurangi [akurasi](/wiki/Accuracy) pemain.
+
+## Argumen
+
+Argumen yang mendukung double tapping biasanya merujuk bahwa cara cepat untuk memungkinkan pemain mempelajari pergerakan beatmap yang muncul setelah pola yang bermasalah, sementara pemain yang menentang double tapping berpendapat bahwa ini adalah cara mudah untuk melewati bagian sulit yang harus dipelajari.
+
```
Comment (ViaSnake) [inline wiki/Play_style/Double_tapping/id.md:22]:
```suggestion

```

**Review by ViaSnake — CHANGES_REQUESTED** [review]

File: wiki/Play_style/Double_tapping/id.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  - tap
+  - hybrid
+  - play style
+---
+
+# Double tapping
+
+*Berbeda konteks dengan [Alternating](/wiki/Play_style/Alternating).*
+
+**Double tapping** adalah gaya tapping untuk [osu!](/wiki/Game_mode/osu!) dimana pemain mengetuk circles, menggeser slider, dan memutar spinner dengan menekan kedua tombol pada keyboard mereka (`Z` dan `X` secara default). Gaya ini paling sering digunakan sebagai pengganti sementara gaya tapping [stacks](/wiki/Mapping_Techniques/Stack) dan [streams](/wiki/Beatmap/Pattern/Stream) yang cepat dan terlalu sulit untuk sebagian pemain. Gaya ini dikenal paling tidak akurat.
```
Comment (mousesalt) [inline wiki/Play_style/Double_tapping/id.md:13]:
```suggestion
**Double tapping** adalah sebuah gaya tapping untuk [osu!](/wiki/Game_mode/osu!) di mana pemain mengetuk circle, menggeser slider, dan memutar spinner dengan menekan kedua tombol pada keyboard mereka (secara umum `Z` dan `X`). Gaya ini paling sering digunakan sebagai pengganti sementara gaya tapping [stack](/wiki/Mapping_Techniques/Stack) dan [stream](/wiki/Beatmap/Pattern/Stream) yang cepat dan terlalu sulit untuk sebagian pemain. Gaya ini dikenal paling tidak akurat.
```

File: wiki/Play_style/Double_tapping/id.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+## Penggunaan
+
+Double tapping biasanya menggantikan gaya tapping utama pemain karena mudah digunakan dan meningkatkan "kecepatan" (meskipun pemain sebenarnya tidak mengetuk lebih cepat). Cara untuk mendapatkan kecepatan yang lebih besar ini dapat dicapai dengan memanfaatkan hit window yang bertumpukan, muncul keseringan pada not-not bertumpuk dan stream. Jadi, satu input tombol terhitung sebagai satu hit circle dalam stack, lingkaran berikutnya juga akan tercatat sebagai hit karena hit window mereka sangat berdekatan sehingga saling tumpang tindih. Tetapi sebagai akibatnya, hampir tidak mungkin untuk mengontrol waktu tapping ini yang dapat mengurangi [akurasi](/wiki/Accuracy) pemain.
```
Comment (mousesalt) [inline wiki/Play_style/Double_tapping/id.md:17]:
```suggestion
Double tapping biasanya menggantikan gaya tapping utama pemain karena mudah digunakan dan meningkatkan "kecepatan" (meskipun pemain sebenarnya tidak mengetuk lebih cepat). Peningkatan kecepatan ini dicapai dengan mengambil keuntungan tumpang tindih dari hit window yang paling sering terjadi pada stack dan stream. Jadi, satu input tombol terhitung sebagai satu hit circle dalam stack, circle berikutnya juga akan terhitung sebagai hit karena hit window mereka sangat berdekatan sehingga saling tumpang tindih. Tetapi sebagai akibatnya, hampir tidak mungkin untuk mengontrol waktu tapping ini yang dapat mengurangi [akurasi](/wiki/Accuracy) pemain.
```

Comment (mousesalt) [reply wiki/Play_style/Double_tapping/id.md:17]: missing several phases on EN
`This increase in speed is achieved by taking advantage of the overlapping hit windows that occur most frequently in stacks and streams.`

File: wiki/Play_style/Double_tapping/id.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+## Argumen
+
+Argumen yang mendukung double tapping biasanya merujuk bahwa cara cepat untuk memungkinkan pemain mempelajari pergerakan beatmap yang muncul setelah pola yang bermasalah, sementara pemain yang menentang double tapping berpendapat bahwa ini adalah cara mudah untuk melewati bagian sulit yang harus dipelajari.
```
Comment (mousesalt) [inline wiki/Play_style/Double_tapping/id.md:21]:
```suggestion
Argumen yang mendukung double tapping biasanya merujuk bahwa cara cepat untuk memungkinkan pemain mempelajari pergerakan beatmap yang muncul setelah pattern yang bermasalah, sementara pemain yang menentang double tapping berpendapat bahwa ini adalah cara mudah untuk melewati bagian sulit yang harus dipelajari.
```

File: wiki/Play_style/Double_tapping/id.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+---
+tags:
+  - keyboard
+  - tap
+  - hybrid
+  - play style
+---
```
Comment (ZeroPyrozen) [inline wiki/Play_style/Double_tapping/id.md:7]: add `- gaya bermain` stub if possible

File: wiki/Play_style/Double_tapping/id.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+**Double tapping** adalah sebuah gaya tapping untuk [osu!](/wiki/Game_mode/osu!) di mana pemain mengetuk circle, menggeser slider, dan memutar spinner dengan menekan kedua tombol pada keyboard mereka (secara umum `Z` dan `X`). Gaya ini paling sering digunakan sebagai pengganti sementara gaya tapping [stack](/wiki/Mapping_Techniques/Stack) dan [stream](/wiki/Beatmap/Pattern/Stream) yang cepat dan terlalu sulit untuk sebagian pemain. Gaya ini dikenal paling tidak akurat.
+
+## Penggunaan
+
+Double tapping biasanya menggantikan gaya tapping utama pemain karena mudah digunakan dan meningkatkan "kecepatan" (meskipun pemain sebenarnya tidak mengetuk lebih cepat). Peningkatan kecepatan ini dicapai dengan mengambil keuntungan tumpang tindih dari hit window yang paling sering terjadi pada stack dan stream. Jadi, satu input tombol terhitung sebagai satu hit circle dalam stack, circle berikutnya juga akan terhitung sebagai hit karena hit window mereka sangat berdekatan sehingga saling tumpang tindih. Tetapi sebagai akibatnya, hampir tidak mungkin untuk mengontrol waktu tapping ini yang dapat mengurangi [akurasi](/wiki/Accuracy) pemain.
```
Comment (ZeroPyrozen) [inline wiki/Play_style/Double_tapping/id.md:17]:
```suggestion
Double tapping biasanya menggantikan gaya tapping utama pemain karena mudah digunakan dan meningkatkan "kecepatan" (meskipun pemain sebenarnya tidak mengetuk lebih cepat). Peningkatan kecepatan ini dicapai dengan mengambil keuntungan tumpang-tindih dari hit window yang paling sering terjadi pada stack dan stream. Jadi, satu input tombol terhitung sebagai satu hit circle dalam stack, circle berikutnya juga akan terhitung sebagai hit karena hit window mereka sangat berdekatan sehingga saling tumpang tindih. Tetapi sebagai akibatnya, hampir tidak mungkin untuk mengontrol waktu tapping ini yang dapat mengurangi [akurasi](/wiki/Accuracy) pemain.
```

**Review by ZeroPyrozen — CHANGES_REQUESTED** [review]

**Review by wowcake — APPROVED** [review]
good to be merged now

**Review by TicClick — APPROVED** [review]

**Pull request merged**